src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,44 @@
+import { FC, SetStateAction } from 'react';
+import { TodoFilter } from '../TodoFilter';
+import { Todo } from '../../types/Todo';
+import { FilterTodosBy } from '../../types/FilterTodosBy';
+
+type Props = {
+  todos: Todo[],
```

Comment:
This type should be `FilterTodosBy` instead of `string` for the `filterBy` prop.